geoh5py/objects/object_base.py

```diff
@@ -247,14 +245,24 @@ def copy(
         if copy_children:
             children_map = {}
             for child in self.children:
-                if isinstance(child, Data) and child.association is not None:
-                    if child.name in ["A-B Cell ID", "Transmitter ID"]:
-                        continue
+                if isinstance(child, Data) and child.association in (
+                    DataAssociationEnum.VERTEX,
+                    DataAssociationEnum.CELL,
+                ):
+                    # if child.name in ["A-B Cell ID", "Transmitter ID"]:
+                    #     continue
+                    #
+                    # child_mask = mask
+                    # if child.association is DataAssociationEnum.CELL:
+                    #     if self.cells is not None and mask is not None:
+                    #         child_mask = np.all(mask[self.cells], axis=1)
+                    # elif child.association is not DataAssociationEnum.VERTEX:
+                    #     child_mask = None
```

Comment:
**suggestion (clean-code):** remove code commented out?

Reply:
Done. Thanks for that.